draft-ietf-tls-tls13.md

```diff
@@ -3159,37 +3165,41 @@ The exact source of each of these secrets depends on the operational
 mode (DHE, ECDHE, PSK, etc.) and is summarized in the table below:
 
 ~~~
-    Key Exchange            Static Secret (SS)    Ephemeral Secret (ES)
-    ------------            ------------------    ---------------------
-    (EC)DHE                   Client ephemeral         Client ephemeral
-    (full handshake)       w/ server ephemeral      w/ server ephemeral
+    Key Exchange     Static Secret (SS)    Ephemeral Secret (ES)
+    ------------     ------------------    ---------------------
+    1-RTT (EC)DHE       N/A (SS = ES)        Client Ephemeral
+                                           + Server Ephemeral
```

Comment:
It's not applicable in that there is no separate value; it's equal to the other. I guess this would come across better with just "SS = ES". The issue I have with the current version is that it's verbose and not clear that these are generated from the same values. (though, the secrets themselves are different)

The "+" could be a "&" instead.